api.ts

```diff
@@ -1151,6 +1172,27 @@ export interface SyncFormulaDef<
    * @hidden
    */
   updateOptions?: Pick<CommonPackFormulaDef<ParamDefsT>, 'extraOAuthScopes'>;
+
+  /**
+   * The javascript function that implements fetching permissions for a set of objects
+   * if the objects in this sync table have permissions in the external system.
+   *
+   * TODO(sam): Unhide this
+   * @hidden
+   */
+  executeGetPermissions?(
+    rows: Array<ObjectSchemaDefinitionType<K, L, SchemaT>>,
```

Comment:
Should we go with a wrapping object here. Might better fit the pattern and leave this more open for modifications in the future

Reply:
Yep had made a change here to a request object